rrfs-test/validated_yamls/templates/obtype_config/ztd_ukmo.yaml

```diff
@@ -0,0 +1,70 @@
+     - obs space:
+         name: Groundgnss  
+         distribution:
+           name: "@DISTRIBUTION@"
+           halo size: 100e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: single_ztd_obs_2024052700.nc
+#            obsfile: ioda/2024052700.rap.t00z.ztd.tm00.bufr_d.nc
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: ztd_single_diag_out_2024052700.nc4
+#-------------------------------------------------------
+#         io pool:
+## to be consistent with the ntask number
+#           max pool size: 120
+#           write multiple files: true
+#-------------------------------------------------------
+
+         simulated variables: [zenithTotalDelay]
+         observed variables: [zenithTotalDelay]
+
+       obs operator:
+         name: GroundgnssMetOffice
+#        variables:
+#        - name: zenithTotalDelay
+
+       linear obs operator:
+         name: GroundgnssMetOffice
+#        variables:
+#        - name: zenithTotalDelay
+
+         observation alias file: obsop_name_map.yaml
+#------------------------------------------------
+         vertical coordinate: height
+         observation vertical coordinate: height
+         interpolation method: linear
+#or ---------------------------------------------
+       obs error:
+         covariance model: diagonal
+
+       obs localizations:
+         - localization method: Horizontal Gaspari-Cohn
+           lengthscale: 300e3 # orig
+
+       obs filters:
+       - filter: PreQC
+         maxvalue: 10
```

Comment:
1. For the entire `obs filters` section, it would be helpful to have some useful comments before each QC filter. For example, this first one could be like
```yaml
       # Reject all obs with QualityMarker > 10
       - filter: PreQC
         maxvalue: 10
```

The comments add visual breaks and just make it easier for someone to know what is going on. For example, the "online domain check" sometimes uses the `BoundsCheck` which can be confusing. Feel free to look at the mesonet and metar yamls for how that has been done.

2. Is `PreQC` a filter? What is it checking? To my knowledge, `PreQC` has been replaced with `QualityMarker` (although there are a lot of places in JEDI that still use `PreQC`). This section could likely also be done like:
```yaml
         # Reject all obs with QualityMarker > 3
         - filter: RejectList
           apply at iterations: 0,1
           where:
           - variable: QualityMarker/zenithTotalDelay
             is_in: 4-15
```
3. I think we typically reject obs with `PreQC`/`QualityMarker` > 3. Is there any reason for setting 10 as the maxvalue? I'll caveat this with: I'm not sure how robust the `QualityMarker` values are as I think those are very crude QC mainly for a time when there wasn't as robust QC checks as there are today.

Reply:
Thanks for the comments. 
1. Will add these comments. 
2. The PreQC filter exists and works well as described in JEDI doc 8.0, which is similar to GSI. 
3.  The actual QC values are not established yet (still missing) yet in the Bufr dataset. Here only a trial (0-10) is used here temporarily for testing.